data/CMakeLists.txt

```diff
@@ -180,6 +180,21 @@ ADD_DEPENDENCIES(testdata data)
 SET(ARCHIVE_NAME {ANAME})
 SET(ARCHIVE_FILES {AFILES})
 
+#-----------------------------------------------------------------------------
+# CMake target to handle make-time requests to build targets that require
+# 7z archiver yet none was provided. The test "" forces make-time failure
+# whereas test " " (note the space) silently succeeds.
+#-----------------------------------------------------------------------------
+IF(NOT DEFINED ARCHIVER_CMD)
+    ADD_CUSTOM_TARGET(missing-7z-archiver ALL
+        COMMAND ${CMAKE_COMMAND} -E echo ""
+        COMMAND ${CMAKE_COMMAND} -E echo "No 7z archiver"
+        COMMAND ${CMAKE_COMMAND} -E echo ""
+        COMMAND test "")
```

Comment:
@biagas a key concern here is whether this line to run the `test` command (and on line 195), works ok on windows. I think it should (windows power shell) but honestly don't know. The intention is to either force a failure of this target (line 193) or silently succeed (line 195)